css/elements.css

```diff
@@ -548,3 +548,41 @@ tr.ins > td > ins, li.ins > ins {
     margin: 0; padding: 0;
   }
 }
+
+/* extra styles for oldtoc */
+body.oldtoc .anchor {
+  display: none;
+}
+
+body.oldtoc emu-annex h1 {
```

Comment:
the following 3 rules are just to add the word Annex as a prefix, and centering the secnum. We could probably do this when generating the secnum though.